pipeline/merger/src/main/resources/application.conf

```
@@ -1,6 +1,7 @@
 aws.metrics.namespace=${?metrics_namespace}
 aws.sqs.queue.url=${?merger_queue_id}
-aws.message.writer.sns.topic.arn=${?merger_topic_arn}
-aws.message.writer.s3.bucketName=${?messages_bucket_name}
 aws.vhs.s3.bucketName=${?vhs_recorder_bucket_name}
 aws.vhs.dynamo.tableName=${?vhs_recorder_table_name}
+aws.message.writer.s3.bucketName=${?messages_bucket_name}
+work_sender.aws.message.writer.sns.topic.arn=${?merger_works_topic_arn}
+image_sender.aws.message.writer.sns.topic.arn=${?merger_images_topic_arn}
```

```suggestion
workSender.aws.message.writer.sns.topic.arn=${?merger_works_topic_arn}
imageSender.aws.message.writer.sns.topic.arn=${?merger_images_topic_arn}
```

We don't really use snake case anywhere in these files, pedantry maybe..

---

:+1: although given [this] I think I'll change to kebab case? https://github.com/lightbend/config/blob/master/HOCON.md#hyphen-separated-vs-camelcase

---

Didn't know it was called `kebab-case`, explains a lot as I love kebab case and also kebabs

---

Loads of meaning into grammatical characters 💥 !